core/jvm/src/main/scala/reftree/util/Reflection.scala

```diff
@@ -1,12 +1,19 @@
 package reftree.util
 
-object Reflection {
+private[reftree] object Reflection {
```

Comment:
> Is it to avoid exposing it as a public API from this library?

Yeah, that is precisely my point. Since we have package shadowing within the `HackedCollectionInstances`, we have to keep this code duplicated. I'm not happy about this violation of DRY, but having that unsafe utility public is unsound to me. But I'm not standing solid on this and could revert changes if you wish.